src/server/pegasus_server_impl.cpp

```diff
@@ -3007,6 +3013,97 @@ void pegasus_server_impl::reset_usage_scenario_options(
     target_opts->max_write_buffer_number = base_opts.max_write_buffer_number;
 }
 
+void pegasus_server_impl::recalculate_data_cf_options(
+    const rocksdb::ColumnFamilyOptions &cur_data_cf_opts)
+{
+#define UPDATE_OPTION_IF_NEEDED(option, value, str_value)                                          \
+    if ((value) != cur_data_cf_opts.option) {                                                      \
+        new_options[#option] = (str_value);                                                        \
+    }
+
+    if (!_is_need_update_data_cf_opts)
```

Comment:
What's the aim of this variable?

Reply:
Avoid `recalculate_data_cf_options` being called all the time